<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>İDDDA ANALİZ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="rootTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1291</width>
      <height>771</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string extracomment="hello"/>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="teamAnalysisTab">
     <attribute name="title">
      <string>Takım Analizi</string>
     </attribute>
     <widget class="QTabWidget" name="teamAnalysisTabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1281</width>
        <height>741</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="teamStatsTab">
       <attribute name="title">
        <string>Takım İstatistiği</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>381</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="teamAnalysisLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Takım listesindeki isimlerden biri olmalı!</string>
           </property>
           <property name="text">
            <string>Takım İsmi :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="teamAnalysisLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="teamAnalysisButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>Tıklayınca sonuçlar görünür.</string>
           </property>
           <property name="text">
            <string>İSTATİSTİKLERİ</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="betHistoryButton">
        <property name="geometry">
         <rect>
          <x>890</x>
          <y>30</y>
          <width>179</width>
          <height>29</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Tıklayınca sonuçlar görünür.</string>
        </property>
        <property name="text">
         <string>GEÇMİŞ BAHİSLERİN</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="teamStatsTextArea">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>71</y>
          <width>671</width>
          <height>621</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="betHistoryTextArea">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>70</y>
          <width>601</width>
          <height>621</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="bestTeamsTab">
       <attribute name="title">
        <string>En İyi Takımlar</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>448</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="bestTeamLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Sıralama Çeşidi :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="bestTeamComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>En İyi Sonuç Alanlar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>En Kötü Sonuç Alanlar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>En Çok Gol Atanlar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>En Az Gol Atanlar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>En Çok Gol Yiyenler</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>En Az Gol Yiyenler</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Maçlarında En Çok Gol Olanlar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Maçlarında En Az Gol Olanlar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Maçlarında En Çok Geri Dönüş Yaşayanlar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>En Çok Korner</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="bestTeamsTextArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>1141</width>
          <height>641</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="compareTwoTab">
     <attribute name="title">
      <string>Takım Karşılaştırma</string>
     </attribute>
     <widget class="QPushButton" name="compareButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Karşılaştır</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>831</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="homeTeamLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ev Sahibi :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="homeTeamLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="awayTeamLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Deplasman :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="awayTeamLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="compTwoLeagueLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Hangi Maçlara Göre?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="compTwoComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Sadece Lig</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sadece Şamp. Ligi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Her İkisi de</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="compTextArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>1141</width>
        <height>641</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="leagueAnalysisTab">
     <attribute name="title">
      <string>Lig Analizi</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>20</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="leagueLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Lig Seç :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="leagueComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Almanya</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>İngiltere</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>İspanya</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>İtalya</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Türkiye</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="teamListTab">
     <attribute name="title">
      <string>Takım Listesi</string>
     </attribute>
     <widget class="QTextBrowser" name="teamListTextArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1141</width>
        <height>751</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="prevBetsTab">
     <attribute name="title">
      <string>Geçmiş Bahisler</string>
     </attribute>
     <widget class="QTabWidget" name="prevBetsTabInner">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>9</y>
        <width>1271</width>
        <height>731</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="betDateSearchTab">
       <attribute name="title">
        <string>Zaman Ara</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>621</width>
          <height>29</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="startDateLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Başlangıç Tarihi</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="startDateLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="endDateLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Bitiş Tarihi</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="endDateLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dateSearchButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Ara</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="betDateSearchStatTextArea">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>50</y>
          <width>601</width>
          <height>651</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="betDateSearchBetsTextArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>661</width>
          <height>651</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="betWeekTab">
       <attribute name="title">
        <string>Hafta Ara</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>10</y>
          <width>591</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="betWeekLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Bülten Haftası: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="betWeekComboBox"/>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="betWeekBetsTextArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>661</width>
          <height>651</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="betWeekStatsTextArea">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>50</y>
          <width>601</width>
          <height>651</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="betTypeSearchTab">
       <attribute name="title">
        <string>Bahis Türü Ara</string>
       </attribute>
       <widget class="QTextBrowser" name="betTypeStatsTextArea">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>40</y>
          <width>601</width>
          <height>651</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="betTypeBetsTextArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>661</width>
          <height>651</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>591</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="betTypeLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Bahis Türü: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="betTypeComboBox"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="calculatorTab">
     <attribute name="title">
      <string>Sistem Oran Hesaplama</string>
     </attribute>
     <widget class="QPushButton" name="calculateButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hesapla</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="calculateTextArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>721</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>531</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="oddsLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Oranlar: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="oddsLabel_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Örn(1.2,3.5,4.2)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>351</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="whichSystemLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sistem Kaç: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="whichSystemLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="whichSystemLabel_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Örn(2,3,4)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="howManyMatchLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Kaç Maç: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="howManyMatcLine">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1293</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSe_enekler">
    <property name="title">
     <string>Seçenekler</string>
    </property>
    <addaction name="actionBa_ka_Bi_i"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menuSe_enekler"/>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
  <action name="actionBa_ka_Bi_i">
   <property name="text">
    <string>Başka Bişi</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
